modules/openapi-generator/src/main/resources/csharp-netcore/libraries/unityWebRequest/ConnectionException.mustache

```diff
@@ -0,0 +1,21 @@
+{{>partial_header}}
+
+using System;
+using UnityEngine.Networking;
+
+namespace {{packageName}}.Client
+{
+    public class ConnectionException : Exception
+    {
+        public UnityWebRequest.Result Result { get; private set; }
```

Comment:
`UnityWebRequest.Result` doesn't exist in Unity 2019

Reply:
Good callout, we need to decide the Unity version supported.
Right now, the oldest LTS on Unity Hub is 2020 (in the official releases section) this is why I felt this is ok.
Since this is a new generator, I think 2020.3 is a good start. 

If many people feel strongly about this, then we could probably adjust

Reply:
I can understand that call. Speaking from personal experience, this is a useful tool to have but some companies get stuck on a specific Unity version. My current employer is using 2019 and my previous employer was tied to 2018 so while Unity wants to keep people moving forward, the reality is a lot slower in the wild.

Reply:
@iBicha @realworld666 I'll need to disagree with supporting old Unity versions. It increases test burden and with Unity dropping 2020.3 LTS this year 2019 is even more antiquated. If someone needs to use this generator on <2020.3, then let them modify the source themselves, it's not hard.